src/components/DateInput/Day.tsx

```diff
@@ -41,23 +111,41 @@ const Day: React.FC<DayProps> = ({ day, month, year, daySelected, onDaySelect =
   );
 
   const isSelected = React.useCallback(() => {
-    if (!daySelected || !date) {
+    if (!daySelected && !dayRangeSelected) {
+      return false;
+    }
+    if (!date) {
       return false;
     }
 
-    return date.isSame(daySelected, 'date');
-  }, [date, daySelected]);
+    if (daySelected) {
+      return date.isSame(daySelected, 'date');
+    }
+    // @ts-ignore
+    const [start, end] = dayRangeSelected;
+    return (start && date.isSame(start, 'date')) || (end && date.isSame(end, 'date'));
+  }, [date, daySelected, dayRangeSelected]);
+
+  const isWithinRange = React.useCallback(() => {
+    if (!dayRangeSelected) {
+      return false;
+    }
+    // @ts-ignore
+    const [start, end] = dayRangeSelected;
+    return start && date.isAfter(start, 'date') && end && date.isBefore(end, 'date');
+  }, [date, dayRangeSelected]);
 
   return (
-    <MemoCell
-      role="gridcell"
-      aria-disabled="false"
-      aria-selected={isSelected()}
+    <MemoOuterCell
       aria-label={date.format('dd MMM DD YYYY')}
       onClick={onDaySelectClick}
+      role="gridcell"
+      aria-busy={isWithinRange()}
+      aria-selected={isSelected()}
     >
-      {day}
-    </MemoCell>
+      <Box aria-placeholder="true" tabIndex={-1} pointerEvents="none" />
```

Comment:
This is the trailing or the leading highlighted background for the selected dates within the range.
The date gets highlighted as a circle and we shall move the "placeholder accordingly in order to create a stacked  effect.
<img width="1013" alt="Screenshot 2020-09-02 at 2 09 09 PM" src="https://user-images.githubusercontent.com/1022166/91974221-2431b900-ed26-11ea-9b45-86ef07878d70.png">

Reply:
hmm  I'm thinking of ways to make this more evident, since an  `aria-placeholder` on a `Box` is not intuitive that's **only** creating a "stylistic" change.

Can we add  a pseudoelement  on the `Day` component to do that? We could use  `_before` or `_after`